features/step_definitions/files_steps.rb

```diff
@@ -12,8 +12,7 @@
 
 
 Then(/^(?:now I|I still) have the following committed files$/) do |files_data|
-  files_data.map_headers!(&:downcase)
-  verify_files files_data.hashes
+  files_data.diff! files_in_branches
```

Comment:
This is checking that the headers match, it just makes use not to have to write them out.